.gitlab.mk

```diff
@@ -137,21 +140,20 @@ package: deploy_prepare
 	TARBALL_EXTRA_ARGS="--exclude=*.exe --exclude=*.dll"                                                            \
 	PRESERVE_ENVVARS="TARBALL_EXTRA_ARGS,${PRESERVE_ENVVARS}" ./packpack/packpack
 
-# found that libcreaterepo_c.so installed in local lib path
-deploy: export LD_LIBRARY_PATH=/usr/local/lib
-
 deploy:
-	echo ${GPG_SECRET_KEY} | base64 -d | gpg --batch --import || true
-	case "${GITHUB_REF}" in                                       \
-	refs/tags/*-alpha*|refs/tags/*-beta*|refs/tags/*-rc*) \
-	    ./tools/update_repo.sh -o=${OS} -d=${DIST}            \
-			-b="${PRERELEASE_REPO_S3_DIR}/${BUCKET}" build ; \
-	        ;;                                                    \
-	refs/tags/*)                                                  \
-		./tools/update_repo.sh -o=${OS} -d=${DIST}            \
-			-b="${RELEASE_REPO_S3_DIR}/${BUCKET}" build ; \
-	        ;;                                                    \
-	esac
+	if [ -z "${REPO_S3_DIR}" ]; then \
+		echo "Env variable 'REPO_S3_DIR' must be defined!"; \
+		exit 1; \
+	fi; \
+	CURL_CMD="curl \
```

Comment:
I use `-LfsS` curl options in scripts. AFAIS, it is what we want here as well.

> -L, --location
>
> (HTTP) If the server reports that the requested page has moved to a different
> location (indicated with a Location: header and a 3XX response code), this
> option  will  make curl  redo  the  request  on the new place. <...>

> -f, --fail
>
> (HTTP)  Fail  silently  (no  output  at  all) on server errors. This is
> mostly done to enable scripts etc to better deal with failed attempts. In
> normal cases when an HTTP server fails to deliver a document, it returns an
> HTML document stating so (which often also describes why and more). This flag
> will prevent curl from outputting that  and return error 22. 
>
> <...>

> -s, --silent
>
> Silent  or  quiet  mode. Do not show progress meter or error messages.  Makes
> Curl mute. It will still output the data you ask for, potentially even to the 
> terminal/stdout unless you redirect it. 
>
> Use -S, --show-error in addition to this option to disable progress meter but 
> still show error messages.
>
> <...>

> -S, --show-error
>
> When used with -s, --silent, it makes curl show an error message if it fails.
>
> <...>

Reply:
Fixed.